caas/kubernetes/provider/application/application.go

```diff
@@ -242,12 +242,13 @@ func (a *app) Ensure(config caas.ApplicationConfig) (err error) {
 		applier.Apply(&resources.StorageClass{StorageClass: sc})
 		return nil
 	}
-	var configureStorage = func(storageUniqueID string, handlePVC handlePVCFunc) error {
+	var configureStorage = func(storageUniqueID string, handlePVC handlePVCFunc, handlePVCVolume handlePVCVolumeFunc) error {
```

Comment:
I wonder if we can think of a better name for `handlePVCVolume` eg `handleVolumeAttachment`